### PR TITLE
created a docs website

### PR DESCRIPTION
**Pull request by ski043**
# Description

Created a documentation website for remix-auth with strategies. 

used Nextra as a template.

demo: https://remix-docs-flame.vercel.app/

### CODE REVIEW

**Comment by cayter** [comment]
I was searching so hard for this. Any chance we can get this in soon?

**Comment by kevinccbsg** [comment]
This looks amazing @ski043! @sergiodxa would you include this documentation?

File: docs/website/pages/strategies/discord.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+### Install the package
+
+import { Tab, Tabs } from "nextra-theme-docs";
+
+{/* prettier-ignore-start */}
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
+  <Tab>
+    ```bash copy
+    npm install remix-auth-linkedin
```
Comment (nwvh) [inline docs/website/pages/strategies/discord.mdx:33]: I think you forgot to change the package name to `remix-auth-discord`

**Comment by sergiodxa** [comment]
Thanks for this, I ended up using TypeDoc to publish to GitHub Pages a simple API reference docs based on the actual code here https://sergiodxa.github.io/remix-auth/

**Pull request closed without merging**